pkg/setting/setting_test.go

```diff
@@ -389,3 +389,28 @@ func TestAuthDurationSettings(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, maxLifetimeDurationTest, cfg.LoginMaxLifetime)
 }
+
+func TestGetCDNPath(t *testing.T) {
```

Comment:
Would be nice with one test where cdn path includes a path as well, e.g. `http://cdn.grafana.com/something/`

Reply:
:+1: